### PR TITLE
Random Forest Model

### PR DESCRIPTION
**Pull request by adibiasio**
Sampled flight data, preprocessed it, used it to build a random forest model, and fine-tuned it with hyperparameterization.

                  precision    recall  f1-score   support
       CANCELLED       0.87      0.96      0.91     43496
        DEL_0_30       0.60      0.58      0.59     43949
       DEL_30_60       0.70      0.75      0.73     43477
          DEL_60       0.85      0.85      0.85     43334
        DIVERTED       0.65      0.43      0.52     43438
          ONTIME       0.61      0.74      0.67     43471
        accuracy                           0.72    261165
       macro avg       0.71      0.72      0.71    261165
    weighted avg       0.71      0.72      0.71    261165

### CODE REVIEW

**Review by ethanm88 — APPROVED** [review]
Looks good!
